src/uu/ls/src/ls.rs

```diff
@@ -2096,19 +2111,35 @@ fn format_prefixed(prefixed: NumberPrefix<f64>) -> String {
     }
 }
 
-fn display_size_or_rdev(metadata: &Metadata, config: &Config) -> String {
-    #[cfg(unix)]
+#[allow(dead_code)]
```

Comment:
This isn't necessary

Reply:
I think we need to include something, as clippy will warn on no devices on Windows.

Reply:
Ah it's a window-specific thing, that explains why I didn't see the error. You could also mark the enum variant with `cfg(unix)`, I think.

Reply:
Nevermind, then you're gonna have to do that everywhere, that's not practical. `allow(dead_code)` is fine then